devicetypes/smartthings/leaksmart-water-sensor.src/leaksmart-water-sensor.groovy

```diff
@@ -0,0 +1,136 @@
+		capability "Battery"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Water Sensor"
+		capability "Temperature Measurement"
+
+		fingerprint inClusters: "0000,0001,0003,0402,0B02,FC02", outClusters: "0003,0019", manufacturer: "WAXMAN", model: "leakSMART Water Sensor V2", deviceJoinName: "leakSMART Water Sensor"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "water", type: "generic", width: 6, height: 4) {
+			tileAttribute ("device.water", key: "PRIMARY_CONTROL") {
+				attributeState("wet", label:'${name}', icon:"st.alarm.water.wet", backgroundColor:"#3277e5")
+				attributeState("dry", label:'${name}', icon:"st.alarm.water.dry", backgroundColor:"#edbd00")
+			}
+		}
+		valueTile("temperature", "device.temperature", width: 2, height: 2) {
+			state("temperature", label: '${currentValue}°',
+					backgroundColors: [
+							[value: 31, color: "#153591"],
+							[value: 44, color: "#1e9cbb"],
+							[value: 59, color: "#90d2a7"],
+							[value: 74, color: "#44b621"],
+							[value: 84, color: "#f1d801"],
+							[value: 95, color: "#d04e00"],
+							[value: 96, color: "#bc2323"]
+					])
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label:'${currentValue}% battery', unit:""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+		main "water"
+		details(["water", "temperature", "battery", "refresh"])
+	}
+}
+
+private getBATTERY_PERCENTAGE_REMAINING() { 0x0021 }
+private getTEMPERATURE_MEASURE_VALUE() { 0x0000 }
+private getEVENTS_ALERTS_CLUSTER() { 0x0B02 }
+
+def installed() {
+	sendEvent(name: "water", value: "dry", displayed: false)
+	refresh()
+}
+
+def parse(String description) {
+	def map = zigbee.getEvent(description)
+	if(!map) {
+		map = parseAttrMessage(description)
+	} else if (map.name == "temperature") {
+		if (tempOffset) {
+			map.value = (int) map.value + (int) tempOffset
+		}
+		map.descriptionText = temperatureScale == 'C' ? "${device.displayName} was ${map.value}°C" : "${device.displayName} was ${map.value}°F"
+		map.translatable = true
+	}
+
+	def result = map ? createEvent(map) : [:]
+
+	if (description?.startsWith('enroll request')) {
+		def cmds = zigbee.enrollResponse()
+		result = cmds?.collect { new physicalgraph.device.HubAction(it)}
+	}
+	log.debug "Description ${description} parsed to ${result}"
+	return result
+}
+
+private Map parseAttrMessage(description) {
+	def descMap = zigbee.parseDescriptionAsMap(description)
+	def map = [:]
+	if(descMap?.clusterInt == EVENTS_ALERTS_CLUSTER) {
+		map = descMap?.data[1] == "81" ? getWaterDetection(descMap?.data[2]) : [:]
```

Comment:
Is this value `0x81` and the value below for water `0x11` manufacturer specific?